k8s/conbench-grafana-dashboard-configmap.template.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  conbench-grafana-dashboard.json: |
+    <CONBENCH_GRAFANA_DASHBOARD_JSON>
+metadata:
+  labels:
+    app.kubernetes.io/component: grafana
+    app.kubernetes.io/name: grafana
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 9.3.2
+    grafana_dashboard: "1"
```

Comment:
The `grafana_dashboard: "1"` technique... is not needed I think. Will have to revisit that. For now, things are working.